bzira.py

```diff
@@ -370,14 +355,13 @@ def proces(bug, bugs):
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+# use chfieldfrom instead of last_change_time so that the URL shown with buglist.cgi returns the same results as the xmlrpc.cgi query
+print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
You may recall this fix from #17 when I tried to convince you that a valid, clickable console URL was a good idea. Well, it's back. And it's better than ever.